src/controls/mdfbPopupLink.js

```diff
@@ -23,7 +23,8 @@ module.exports = function ($compile, $mdDialog, $injector) {
         $mdDialog.show({
           contentElement: '#popup_' + scope.field.name,
           targetEvent: ev,
-          clickOutsideToClose: true
+          clickOutsideToClose: true,
+          fullscreen: true
```

Comment:
Should popups always be fullscreen? Maybe only when a particular breakpoint is set?

Reply:
This setting only uses the full page width/height on smaller windows, it doesnt take all the space by default